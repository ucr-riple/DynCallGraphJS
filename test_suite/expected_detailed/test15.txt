{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test15.js:16:5:16:8)': [
        'g (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test15.js:5:1:7:2)'
    ]
}