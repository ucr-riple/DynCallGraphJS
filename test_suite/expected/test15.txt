{ '(test15.js:16:5:16:8)':
   [ '(test15.js:5:1:7:2)' ] }